gnomad/sample_qc/ancestry.py

```diff
@@ -163,19 +163,17 @@ def assign_population_pcs(
 
     hail_input = isinstance(pop_pca_scores, hl.Table)
     if hail_input:
```

Comment:
I'm thinking we might want to add a check that this is a list of integers with an error message detailing the expectation